src/components/control/autocompleter.jsx

```diff
@@ -1,28 +1,45 @@
 import PropTypes from 'prop-types';
-import React, { forwardRef, useEffect, useState } from 'react';
+import React, { forwardRef, useEffect, useImperativeHandle, useRef, useState } from 'react';
 import useFetch from '@bloodyaugust/use-fetch';
 import useMounted from '../../hooks/use-mounted.js';
 import Select from './select.jsx';
 import ListOption from '../list-option/list-option.jsx';
 import { API_ROOT } from '../../mocks/mock-constants.js';
 
 const Autocompleter = forwardRef(function Autocompleter(props, ref) {
-  const { execute } = useFetch();
+  const { execute: executeModels } = useFetch();
+  const { execute: executeValue } = useFetch();
```

Comment:
pedantic: `executeModels` sounds like murdering people that get paid to show clothing lines. I'd recommend `fetchModels` personally.

Also, I'm not 100% sure I understand why we need a separate hook for each `fetch` request - that seems super odd to me imo

Reply:
`fetchModels` is already defined in this scope :( I had to switch it. 

I think we double `useFetch` since that's part of its API. I don't recall the details but it needs to generate a function with some variables associated with it. Kinda like state but not really. 🤔

Reply:
Most of the state is hidden inside the native `fetch`, but there is some relevant state as well that's part of the hook. Overall I think it makes much more sense for a hook-based fetching API to handle a single request and not multiples. You can re-use the same hook, but basically we want to be able to manage them individually (like cancelling), and the interface for handling multiple gets really messy really fast. Maybe a good topic of discussion for arch meeting?